libs/utils/decorate.js

```diff
@@ -96,3 +96,22 @@ export function decorateTextOverrides(el, options = ['-heading', '-body', '-deta
     el.classList.remove(override);
   });
 }
+
+export function decorateIconStack(el) {
+  const ulElems = el.querySelectorAll('ul');
+  if (ulElems.length) {
+    const stackEl = ulElems[ulElems.length - 1];
+    stackEl.classList.add('icon-stack-area', 'body-s');
+    el.classList.add('icon-stack');
+    const items = stackEl.querySelectorAll('li');
+    [...items].forEach((i) => {
+      const links = i.querySelectorAll('a');
+      if (i.querySelectorAll('a').length <= 1) return;
```

Comment:
```suggestion
      const links = i.querySelectorAll('a');
      if (links.length <= 1) return;
```

Reply:
@chrischrischris I have made the requested changes.